### DISCUSSION TITLE
Is this project the same as the one currently installed with Windows?

### DISCUSSION THREAD

**Discussion by ketozhang** [brettcannon/python-launcher#360] [Q&A] [answered]
Windows currently have this available to them:
https://docs.python.org/3/using/windows.html#from-the-command-line

It's not clear to me whether this is the same despite having the same command name `py`.

**Comment by brettcannon** [comment, marked as answer]
They are not exactly the same. How they differ is outlined at https://python-launcher.app/#why-this-tool-exists .

**Reply by ketozhang** [reply]
Home page was updated to address the difference more explicitly. Thank you! https://github.com/brettcannon/python-launcher/pull/272
